nx.json

```diff
@@ -358,7 +358,7 @@
       "implicitDependencies": ["fluid-button"]
     },
     "shared-data-access-strapi": {
-      "tags": ["scope:cms", "type:shared"]
+      "tags": ["scope:cms", "type:shared", "type:library"]
```

Comment:
is the scope cms needed for a lint rule? cannot see the benefit of it currently.